solr_conf/conf/schema.xml

```diff
@@ -8,37 +8,61 @@
 
   <fields>
     <field name="id" type="string" stored="true" indexed="true" multiValued="false" required="true"/>
+    <!-- DEPRECATED: "long" is a Trie based field type; use solr.LongPointField class instead ... -->
```

Comment:
See https://github.com/sul-dlss/sul-solr-configs/pull/280/files for rationale behind schema.xml changes